src/test/java/tests/ForgotPasswordTests.java

```diff
@@ -38,4 +39,19 @@ public void tc09_UserCanResetPassword() {
         LogHelper.info("Describe the issue");
         Assert.fail("Server Error appears with message: Mailbox unavailable. The server response was: that smtp username is not allowed to send");
     }
+
+    @Test(description = "User can't reset password if enter incorrect email address")
+    public void tc10_ResetPasswordWithEnteringIncorrectEmail() {
+        LogHelper.info("Go to Login page");
+        loginPage.goToLoginPage();
+
+        LogHelper.info("Click on 'Forgot password page' link");
+        forgotPasswordPage.clickForgotPasswordLink();
```

Comment:
![image](https://user-images.githubusercontent.com/23515968/132199467-9b823429-c2cd-4e7e-9225-e59e53ba0aea.png)

clickForgotPasswordLink method belongs to LoginPage class, not ForgotPasswordPage class

Reply:
Please update for tc09 also

Reply:
I moved it to LoginPage class. Please help me check it again, thanks